app/controllers/claims/support/schools/mentors_controller.rb

```diff
@@ -1,7 +1,26 @@
 class Claims::Support::Schools::MentorsController < Claims::Support::ApplicationController
   include Claims::BelongsToSchool
+  before_action :set_mentor, only: %i[show remove destroy]
 
   def index
     @pagy, @mentors = pagy(@school.mentors.order(:first_name, :last_name))
   end
+
+  def show; end
+
+  def remove; end
+
+  def destroy
+    mentor_membership = MentorMembership.find_by(mentor_id: @mentor.id, school_id: @school.id)
```

Comment:
```suggestion
    mentor_membership = @mentor.mentor_memberships.find_by(school: @school)
```

A slightly tidier way to write this.